test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -1,9559 +1,9485 @@
-                "compile": {
-                    "System.Runtime.CompilerServices.VisualC.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.Reference/4.2.2.0": {
-                "compile": {
-                    "System.Runtime.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.Extensions.Reference/4.2.2.0": {
-                "compile": {
-                    "System.Runtime.Extensions.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.Handles.Reference/4.1.2.0": {
-                "compile": {
-                    "System.Runtime.Handles.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.InteropServices.Reference/4.2.2.0": {
-                "compile": {
-                    "System.Runtime.InteropServices.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.InteropServices.RuntimeInformation.Reference/4.0.4.0": {
-                "compile": {
-                    "System.Runtime.InteropServices.RuntimeInformation.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.InteropServices.WindowsRuntime.Reference/4.0.4.0": {
-                "compile": {
-                    "System.Runtime.InteropServices.WindowsRuntime.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.Intrinsics/4.0.1.0": {
-                "compile": {
-                    "System.Runtime.Intrinsics.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.Loader.Reference/4.1.1.0": {
-                "compile": {
-                    "System.Runtime.Loader.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.Numerics.Reference/4.1.2.0": {
-                "compile": {
-                    "System.Runtime.Numerics.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.Serialization/4.0.0.0": {
-                "compile": {
-                    "System.Runtime.Serialization.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.Serialization.Formatters/4.0.4.0": {
-                "compile": {
-                    "System.Runtime.Serialization.Formatters.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.Serialization.Json.Reference/4.0.5.0": {
-                "compile": {
-                    "System.Runtime.Serialization.Json.dll": {}
-                },
-                "compileOnly": true
-            },
-            "System.Runtime.Serialization.Primitives.Reference/4.2.2.
```

Comment:
@pragnagopa can we avoid the whitespace changes? that made the diff extremely large. Also looks like the deps file you've changed was created in debug mode.

Reply:
Copied deps file for Release build. Thanks! Can you please take a look at https://github.com/Azure/azure-functions-host/pull/7005/files?w=1 

I tried manually editing the the file but there are quite a few changes , copied the build output to avoid other mistakes.

Reply:
this is fixed now. Thanks!